dataplane/standalone/sai/fdb.cc

```diff
@@ -233,6 +233,9 @@ sai_status_t l_create_fdb_entries(uint32_t object_count,
     LOG(ERROR) << status.error_message();
     return SAI_STATUS_FAILURE;
   }
+  if (object_count != resp.resps().size()) {
```

Comment:
Are the following .cc files code auto-generated?

Reply:
yes